### ISSUE TITLE
MAS4.3.1 Highcontrast white mode: Dropdown "button" isn't visible

### ISSUE THREAD

**Issue by yaananth** [OfficeDev/office-ui-fabric-react#4421]
Goto https://developer.microsoft.com/en-us/fabric#/components/dropdown 

**High contrast white:**

_Actual:_
![image](https://user-images.githubusercontent.com/12280011/38156510-6ca7419a-344c-11e8-8dc4-f13c0fc25fe4.png)

_Expected:_
The dropdown button should be of - white color with violet background.

**High contrast black (no issue here):**

_Actual and Expected:_
![image](https://user-images.githubusercontent.com/12280011/38156521-82d71224-344c-11e8-966a-2e8f74c3b305.png)

The dropdown button should be of - black color with blue background.

**Comment by michaelangotti** [comment]
Bug is already referenced in #4373.  Will close this duplicate.
